docs/data/charts/pyramid-demo/PyramidInverted.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { Unstable_FunnelChart as FunnelChart } from '@mui/x-charts-pro/FunnelChart';
+
+export default function PyramidInverted() {
+  return (
```

Comment:
It would be nice to have a toggle to switch between increasing/decreasing so users can easily compare the difference